### PR TITLE
Russian translation

### PR DESCRIPTION
**Pull request by ghost**
*PR was retracted by author*

### CODE REVIEW

File: src/main/resources/assets/notenoughcreativity/lang/ru_ru.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+    "notenoughcreativity.title": "Творческий инвентарь+",
```
Comment (unascribed) [inline src/main/resources/assets/notenoughcreativity/lang/ru_ru.json:2]: The placement of the + here seems wrong. "инвентарь" appears to be "Inventory", and the plus is attached to "Творческий" elsewhere ("Creative").

File: src/main/resources/assets/notenoughcreativity/lang/ru_ru.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    "notenoughcreativity.ability.health.name": "Предоставить урон",
+    "notenoughcreativity.ability.health.desc": "Предоставляет тебе урон и показывает полоску здоровья. Ты по-прежнему не можешь умереть, за исключением обычных вещей, которые могут убить игроков в творческом режиме.",
+    "notenoughcreativity.ability.instabreak.name": "Без замедления в ломании",
+    "notenoughcreativity.ability.instabreak.desc": "Отключает творческое замедление в ломании, позволяя ломать блоки, как золотая кирка сквозь незерак.",
+    "notenoughcreativity.ability.pickswap.name": "Замена-сбор",
+    "notenoughcreativity.ability.pickswap.desc": "Перемещает уникальные предметы в верхний инвентарь, вместо их удаления, при блокировки сбора с удерживающим предметом. Также удаляет первый совпадающий предмет из верхнего инвентаря при блокировки сбора.",
+    "notenoughcreativity.ability.nightvision.name": "Ночное зрение",
+    "notenoughcreativity.ability.nightvision.desc": "Делает все блоки полнояркими. Не срабатывает при эффекте Ночное зрение.",
+    "notenoughcreativity.ability.noclip.name": "Без быстроты",
```
Comment (unascribed) [inline src/main/resources/assets/notenoughcreativity/lang/ru_ru.json:48]: Multiple machine translators and dictionaries suggest this means "No Speed". NoClip refers to disabling *collision*. Does context really allow this to mean "no collision"?

**Review by exaskye — COMMENTED** [review]
—

**Comment by unascribed** [comment]
??

**Comment by DrHesperus** [comment]
If the author asks questions, then I won't prove anything, if you want, you can translate the mod into Russian yourself.

**Comment by unascribed** [comment]
Okay. If you're going to be a crybaby about your PR *being reviewed in good faith*, I don't want your contributions anyway. Get lost.

**Pull request closed without merging**